libclamav/ole2_extract.c

```diff
@@ -1917,7 +2608,11 @@ cl_error_t cli_ole2_extract(const char *dirname, cli_ctx *ctx, struct uniq **fil
         cli_dbgmsg("OLE2: no VBA projects found\n");
         /* PASS 2/B : OTF scan */
         file_count = 0;
-        ret        = ole2_walk_property_tree(&hdr, NULL, 0, handler_otf, 0, &file_count, ctx, &scansize2);
+        if (bVelvetSweatshop) {
+            ret = ole2_walk_property_tree(&hdr, NULL, 0, handler_otf_velvetsweatshop, 0, &file_count, ctx, &scansize2, &key);
```

Comment:
we should probably cast `(void *)&key` to be explicit. Not sure if there will be a warning on some compiler if we don't.